wrappers/swift/CMakeLists.txt

```diff
@@ -0,0 +1,25 @@
+CMAKE_MINIMUM_REQUIRED(VERSION 3.16)
```

Comment:
This version is higher than root level and Obj-C wrapper level versions, why?

Reply:
This was done when swift generators were failing to be generated and there was one suggestion to change it to this version.
But i think with xcode generators, this is not required. i can change it back to similar to obj-c.

There is below warning of deprecation though:

_CMake Deprecation Warning at CMakeLists.txt:1 (CMAKE_MINIMUM_REQUIRED):
  Compatibility with CMake < 2.8.12 will be removed from a future version of
  CMake.

  Update the VERSION argument <min> value or use a ...<max> suffix to tell
  CMake that the project does not need compatibility with older versions._